front/app/modules/commercial/content_builder/admin/components/contentBuilderToggle.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
The files for components should start with a capital letter - the file here should be called `ContentBuilderToggle.test.tsx`